server/src/intTest/db/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  local-database:
+    container_name: postgres-common
+    image: postgres:16.3
```

Comment:
Please can we use Postgres version 14.10 here - that's what we're running our AWS workloads on